### PR TITLE
feat(Tebex): add presence

### PR DESCRIPTION
**Pull request by DarkVillager**
## Description 

Added tebex & store's with Buycraft/Tebex in it's domain
## Acknowledgements
- [X] I read the [Presence Guidelines](https://github.com/PreMiD/Presences/blob/main/.github/CONTRIBUTING.md)
- [X] I linted the code by running `yarn format`
- [X] The PR title follows the repo's [commit conventions](https://github.com/PreMiD/Presences/blob/main/.github/COMMIT_CONVENTION.md)

## Screenshots
<details>
<summary> Proof showing the creation/modification is working as expected </summary>

</details>

### CODE REVIEW

File: websites/T/Tebex/metadata.json
Comment (EncryptedDev) [inline websites/T/Tebex/metadata.json:10]: @Bas950

File: websites/T/Tebex/presence.ts
Comment (EncryptedDev) [inline websites/T/Tebex/presence.ts:43]: Sounds really repetitive with `details` possibly `View Page` instead

File: websites/T/Tebex/presence.ts
Comment (EncryptedDev) [inline websites/T/Tebex/presence.ts:63]: Ditto

File: websites/T/Tebex/presence.ts
Comment (EncryptedDev) [inline websites/T/Tebex/presence.ts:73]: Not really needed

File: websites/T/Tebex/presence.ts
Comment (EncryptedDev) [inline websites/T/Tebex/presence.ts:108]:
```suggestion
						label: "Read Docs",
```

File: websites/T/Tebex/presence.ts
Comment (EncryptedDev) [inline websites/T/Tebex/presence.ts:128]: Not really needed

**Review by EncryptedDev — CHANGES_REQUESTED** [review]

File: websites/T/Tebex/metadata.json
Comment (DarkVillager) [reply websites/T/Tebex/metadata.json:10]: @QkeleQ10

**Review by EncryptedDev — APPROVED** [review]
lgtm once bas approves translation

**Review by Bas950 — APPROVED** [review]

**Pull request merged**
